tests/playwright-test/to-have-screenshot.spec.ts

```diff
@@ -555,6 +555,20 @@ test('should fail on same snapshots with negate matcher', async ({ runInlineTest
   expect(result.output).toContain('Expected result should be different from the actual one.');
 });
 
+test('should not fail if --ignore-snapshots is passed', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    ...playwrightConfig({ screenshotsDir: '__screenshots__' }),
+    '__screenshots__/a.spec.js/snapshot.png': redImage,
+    'a.spec.js': `
+      pwt.test('is a test', async ({ page }) => {
+        await expect(page).not.toHaveScreenshot('snapshot.png', { timeout: 2000 });
```

Comment:
I think this does not test anything, since screenshot does not match the red image anyway.